src/components/MissionAPP/MissionShelf.jsx

```diff
@@ -0,0 +1,49 @@
+import { useDispatch, useSelector } from 'react-redux';
+import { useEffect } from 'react';
+import { getMissions, selectMission } from 'features/missions/missionsSlice';
+import { nanoid } from '@reduxjs/toolkit';
+
+const MissionShelf = () => {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getMissions());
+  }, []);
+
+  const { missions, isLoading, error } = useSelector(selectMission);
+
+  const nobooks = (
+    <h3 className="center">No Books Found</h3>
+  );
```

Comment:
- [ ]  Here I guess to stay on the topic, it should be `noMissions `and `No Missions Found `

Reply:
Got catch about the names. Already changed.